src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  padding: 0;
+  margin: 0;
+
+  list-style: none;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:not(:last-child) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
if I change modifier nothing happens(
Reuse styles from Stars task so that modifier functionality work as expected